ci/scripts/website_build.sh

```diff
@@ -60,21 +60,20 @@ main() {
     # Copy the version script and regenerate the version list
     # The versions get embedded into the JavaScript file to save a roundtrip
     rm -f "${site}/version.js"
-    cp "${repo}/docs/source/_static/version.js" "${site}/version.js"
-    echo >> "${site}/version.js"
     echo 'const versions = `' >> "${site}/version.js"
 
     pushd "${site}"
+    rm -f "${site}/versions.txt"
     for inv in */objects.inv; do
+        if [[ "$(dirname $inv)" = "current" ]]; then
+            continue
+        fi
         echo "$(dirname $inv);$(sphobjinv convert json $inv - | jq -r .version)"
     done | sort -t ";" --version-sort | tee --append "${site}/version.js" "${site}/versions.txt"
     popd
 
-    echo '`;' >> "${site}/version.js"
-    git -C "${site}" add -f "version.js"
-
     # Determine the latest stable version
-    local -r latest_docs=$(grep -E ';[0-9]+\.[0-9]+\.[0-9]+$' "${site}/versions.txt" | sort -t ';' --version-sort | tail -n1)
+    local -r latest_docs=$(grep -E ';[0-9]+\.[0-9]+\.[0-9]+$' "${site}/versions.txt" | grep -v -E '^current;' | sort -t ';' --version-sort | tail -n1)
```

Comment:
It seems that this is needless because the above `for inv ...` loop skips `current`.

Reply:
Ah, right.